addons/attach/stringtable.xml

```diff
@@ -175,5 +103,29 @@
       <Russian>Присоединить Ошибка</Russian>
       <Spanish>Error al acoplar</Spanish>
     </Key>
+    <Key ID="STR_ACE_Attach_Item_Attached">
+      <English>%1 Attached</English>
```

Comment:
Insert a line break after the magazine name, as it probably won't be syntactically correct sentence for some languages:
`<English>%1&lt;br/&gt;Attached</English>`